soaking/Makefile

```diff
@@ -0,0 +1,28 @@
+.ONESHELL:
+SHELL := /bin/bash
+.SHELLFLAGS = -ec
+
+ROOT_DIR := $(CURDIR)
+
+NAMESPACE ?= spark-workloads
+
+.PHONY: install
+install:
```

Comment:
should we include an "upgrade" plan too?

Reply:
thanks, made a separate make target for it